app/src/main/java/com/mobile/tandil/javabaseproject/util/Constants.java

```diff
@@ -0,0 +1,6 @@
+package com.mobile.tandil.javabaseproject.util;
+
+public class Constants {
+    public static String CALENDAR_FORMAT = "dd-MM-yyyy HH:mm";
+    public static Integer MINUS_ONE = -1;
```

Comment:
I think this constant could be called  for example `INVALID_PARKING_LOTS` because if you change its value to -5, its name looks wrong

Reply:
done